packages/foreman/rubygem-rkerberos/rubygem-rkerberos.spec

```diff
@@ -0,0 +1,98 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name rkerberos
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.5
+Release: 2%{?dist}
+Summary: A Ruby interface for the the Kerberos library
+Group: Development/Languages
+License: Artistic 2.0
+URL: https://github.com/domcleal/rkerberos
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(rake-compiler)
```

Comment:
I know, this is autogenerated, but I still think we don't need rake-compiler at runtime.

@ekohl can we blacklist it somehow?

Reply:
https://github.com/theforeman/foreman-packaging/blob/rpm/develop/update-requirements does have an NPM blacklist because that's also incomplete. We could implement it there.